portal-wire-test-vectors.md

```diff
@@ -0,0 +1,158 @@
+# Portal Wire Test Vectors
+
+This document provides a collection of test vectors for the Portal wire protocol
+aimed to aid new implementations to conform to the specification.
+
+## Protocol Message Encodings
+
+This section provides test vectors for the individual protocol messages defined
+in the [Portal wire protocol](./portal-wire-protocol.md). These test vectors can
+primarily verify the SSZ encoding and decoding of each protocol message.
+
+### Ping Request
+
+#### Input Parameters
+```
+enr_seq = 1
+data_radius = uint256.high() - 1
```

Comment:
Maybe it's just because Javascript doesn't have a specific type for `uint256` so could we replace `uint256.high()` with `2^256 - 1` or some explicit number?  I coded in `2^256 - 1` using Javascript's native `bigints` and the resulting encoded message I get  is `010101000000000000000c000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff` which is not what you put below.

Reply:
By trial and error, I discovered that `uint256.high()` must be equivalent to 2^256 - 2 in Javascript bigints as that produces the same encoding result you noted below.

Reply:
I assume you mean that `uint256.high() - 1` is equivalent to `2^256 - 2`. Yes, with `high()` I meant the maximum possible number for an uint256, thus all bits set, which equals `2^256 - 1`. I can replace it with that instead to be perfectly clear. I preferred not to use the actual large value.